### PR TITLE
fix: Windows install registers a startup task that never runs

### PR DESCRIPTION
**Pull request by nandyalu**
## The bug

Reported on Discord: after updating to v0.11.4 the web page would not open, `netstat` showed no listening port, and `trailarr logs` said the log file did not exist — while `trailarr status` reported **Ready**.

[`windows.py`](https://github.com/nandyalu/trailarr/blob/dev/scripts/install/platforms/windows.py) interpolated the program path into the Task Scheduler registration without quotes:

```powershell
New-ScheduledTaskAction -Execute {exe} -Argument '"{script}"'
```

`{exe}` is `C:\Program Files\Trailarr\backend\.venv\Scripts\trailarr.exe`, which holds a space. Running the generated command through PowerShell:

```
Execute          : C:\Program
Arguments        : "C:\Program Files\Trailarr\scripts\start\start.py"
WorkingDirectory : Files\Trailarr\backend\.venv\Scripts\trailarr.exe
```

PowerShell binds `C:\Program` to `-Execute` and slurps the remainder into the positional `-WorkingDirectory`. **No error is raised**, so `Register-ScheduledTask` succeeds, the installer's `returncode != 0` check passes, and the install reports success — having registered a task pointing at a file that does not exist.

That accounts for every symptom: the task never runs, so no port is bound and no log file is written; `State` stays `Ready` because the task is registered and idle; and re-installing writes the identical broken task.

## Why it surfaced in v0.11.1+

The quotes came off in `04b3526b` ("fix: minor quotes wrapping") back in v0.9.1. A second bug hid it: `4a7f1633` (v0.9.5) introduced `-DisallowStartIfOnBatteries $false`, which is not a real parameter, so registration failed with exit 1 and the installer aborted **before overwriting the task**. Every user kept their older, correctly quoted task and the app kept working.

`3c398dfe` (v0.11.1) then fixed the battery flags. Registration succeeds again, `Register-ScheduledTask -Force` finally overwrites the good task with the broken one, and the latent bug reached users.

## Changes

- **`scripts/install/platforms/windows.py`** — quote every path; move the command into `build_register_task_ps()` so it can be tested without registering anything. Added a comment on why the quotes are load-bearing, since they were previously removed as cosmetic.
- **`scripts/cli/trailarr_cli.py`** — `trailarr status` now also prints `LastRunTime`, `LastTaskResult` and the action. `State` alone is actively misleading here.
- **`frontend/src/app/auth/login/login.component.ts`** — a network failure (`status 0`) or gateway error (502/503/504) now reports that the server is unreachable instead of "Invalid username or password." The reporter spent their first round of debugging convinced their password was wrong.
- **`backend/tests/scripts/test_windows_task_registration.py`** — 6 regression tests.
- **`docs/release-notes/2026.md`** — three entries under v0.11.5.

## Testing

6/6 new tests pass. Five assert on the generated string; the sixth pipes the real command into PowerShell and asserts the parsed action — it builds the action object only and registers nothing, and is skipped off Windows. That test fails on the old code and passes on the new, so it pins the defect rather than the formatting.

Both edited Python files compile and the new status command parses cleanly. `login.component.ts` typechecks with zero errors, and the existing "shows an error when login fails" spec still passes by construction: it calls `error()` with no argument, so `status` is `undefined`, which is not in the unreachable set and still yields the credentials message.

**Not verified:** the frontend spec could not be executed. `npm run test` is broken repo-wide — `angular.json` is missing the `runner` property that `@angular/build:unit-test` requires — and `tsc` reports 15 pre-existing errors from stale `node_modules`, none in touched files. Both predate this branch. Worth fixing separately so CI can exercise the login change.

## Note for affected users

The fix only applies on the next install. Anyone already broken can repair the existing task in place:

```powershell
$exe    = "C:\Program Files\Trailarr\backend\.venv\Scripts\trailarr.exe"
$script = "C:\Program Files\Trailarr\scripts\start\start.py"
Set-ScheduledTask -TaskName 'Trailarr' -Action (New-ScheduledTaskAction -Execute $exe -Argument "`"$script`"")
Start-ScheduledTask -TaskName 'Trailarr'
```

Re-running the installer before this ships will overwrite that repair.

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

File: scripts/install/platforms/windows.py [hunk 1]
```diff
@@ -17,6 +17,37 @@
...
+$trigger  = New-ScheduledTaskTrigger -AtLogon -User '{username}'
+$settings = New-ScheduledTaskSettingsSet -ExecutionTimeLimit 0 -RestartCount 3 `
+              -RestartInterval (New-TimeSpan -Minutes 1) -StartWhenAvailable `
+              -AllowStartIfOnBatteries -DontStopIfGoingOnBatteries
+$principal = New-ScheduledTaskPrincipal -UserId '{username}' -LogonType S4U -RunLevel Limited
+Register-ScheduledTask -TaskName '{task_name}' `
+  -Description 'Trailarr - Trailer downloader for Radarr and Sonarr' `
+  -Action $action -Trigger $trigger -Settings $settings -Principal $principal -Force | Out-Null
+Start-ScheduledTask -TaskName '{task_name}'
+"""
```
[the remark this answers is not in the document]
Comment (nandyalu) [reply scripts/install/platforms/windows.py:66]: Good catch — fixed in b859f030. A Windows account can genuinely be named `O'Brien`, so this was a real break, not just a hardening concern: registration failed to parse and the installer aborted.

All interpolated values now go through a `_ps_quote()` helper that doubles single quotes. Verified against the PowerShell parser:

```
UNESCAPED -> errors: 1
  The string is missing the terminator: '.
ESCAPED   -> errors: 0
```

Added `test_an_apostrophe_in_the_user_name_is_escaped`, `test_an_apostrophe_in_the_task_name_is_escaped`, and a parametrized parser test covering `someuser`, `First Last` and `O'Brien`. The parser tests parse only — they never register a task.

File: backend/tests/scripts/test_windows_task_registration.py [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+def test_start_script_is_passed_as_a_quoted_argument(build_ps):
+    """The start script path also holds a space, so it stays quoted."""
+    ps = build_ps(EXE, SCRIPT, "someuser")
+    assert f"""-Argument '"{SCRIPT}"'""" in ps
+
+
+def test_task_name_and_user_are_quoted(build_ps):
+    """A user name can hold a space too, and the task name always does not."""
+    ps = build_ps(EXE, SCRIPT, "First Last", task_name="Trailarr")
```
[the remark this answers is not in the document]
Comment (nandyalu) [reply backend/tests/scripts/test_windows_task_registration.py:63]: Agreed, that sentence was garbled. Fixed in b859f030 — it now reads "A user name can hold a space, so it stays inside quotes."

File: scripts/cli/trailarr_cli.py [hunk 3]
```diff
@@ -194,10 +194,16 @@ def _service_status() -> None:
...
+        # last run result and the action tell you whether it ever started.
         subprocess.run(
             ["powershell", "-NonInteractive", "-Command",
-             f"Get-ScheduledTask -TaskName '{_TASK_NAME}' | "
-             f"Select-Object TaskName, State, Description"],
+             f"$t = Get-ScheduledTask -TaskName '{_TASK_NAME}'; "
+             f"$t | Select-Object TaskName, State, Description | Format-List; "
+             f"$t.Actions | Select-Object Execute, Arguments | Format-List; "
+             f"Get-ScheduledTaskInfo -TaskName '{_TASK_NAME}' | "
+             f"Select-Object LastRunTime, LastTaskResult | Format-List"],
```
[the remark this answers is not in the document]
Comment (nandyalu) [reply scripts/cli/trailarr_cli.py:206]: Added `-NoProfile` in b859f030 — worth having, since this command formats output that a profile could alter.

One correction to the rationale though: only 1 of the 5 `powershell` invocations in this file used `-NoProfile` before this change (line 231). The other four — lines 144, 161, 201 and 541 — pass `-NonInteractive` alone, so the prevailing convention here was actually the opposite. I have only changed the line this PR already touches; bringing the remaining three into line feels like a separate cleanup rather than something to fold into a bug fix.
